src/test/java/uk/gov/hmcts/reform/adoption/adoptioncase/caseworker/event/CaseworkerAmendApplicantTest.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.adoption.adoptioncase.caseworker.event;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.ccd.sdk.ConfigBuilderImpl;
+import uk.gov.hmcts.ccd.sdk.api.Event;
+import uk.gov.hmcts.reform.adoption.adoptioncase.event.EventTest;
+import uk.gov.hmcts.reform.adoption.adoptioncase.model.CaseData;
+import uk.gov.hmcts.reform.adoption.adoptioncase.model.State;
+import uk.gov.hmcts.reform.adoption.adoptioncase.model.UserRole;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.adoption.adoptioncase.caseworker.event.CaseworkerAmendApplicant.CASEWORKER_AMEND_APPLICANT;
+
+/**
+ * Unit Test class targeted for Amend Case Details functionality.
+ * CaseworkerAmendApplicant configuration class will be used as base in all Tests
```

Comment:
Are these 2 classes really "used as a base in all Tests"?  I see a lot of tests extending EventTest but not these?

Also given Java doesn't support multi inheritance (excluding interfaces), presumably having this comment on this and CaseworkerAmendCaseTest means both can't be accurate?  Unless are they getting added somewhere I've missed?